<launch>
  <node pkg="tamagawa_imu_driver" name="tag_serial_driver" type="tag_serial_driver">
    <param name="port" value="/dev/imu" />
    <param name="imu_frame_id" value="tamagawa/imu_link" />
  </node>
</launch>
